hil-test/tests/delay.rs

```diff
@@ -1,5 +1,9 @@
 //! Delay Test
 
+//% CHIPS: esp32 esp32c2 esp32c3 esp32c6 esp32s3
+// TODO: esp32h2 is disabled due to https://github.com/esp-rs/esp-hal/issues/1509
```

Comment:
I don't think we need a TODO comment and an issue, we've been working on eliminating TODO comments.